Tutorial/WindowsVersion/tutorial.md

```diff
@@ -0,0 +1,39 @@
+1. Download SPTAG.
+    - git clone https://github.com/microsoft/SPTAG.git
+2. Build SPTAG. The build targets, IndexBuilder.exe, Server.exe, SPTAG.py, SPTAGClient.py, _SPTAG.pyd, and _SPTAGClient.pyd are all we need.
```

Comment:
can you use same rich bullet instead of plain text.... make the markdown looks more easy to read.

Reply:
Done